### ISSUE TITLE
Y24-176 - [BUG] When updating a run under certain conditions the well positions are incorrectly recorded.

### ISSUE THREAD

**Issue by BenTopping** [sanger/traction-ui#1801]
**Describe the bug**
Craig C from slack:
The particular plate ID: 1021188000348050043720241008 was originally used on run 1354. Run 1354 failed over the weekend leaving the box with the first 2 wells untouched. We tried to create a new run to include this plate but experienced difficulty. We were having trouble creating a new run with this plate ID whilst adding libraries to wells C01 and D01 so edited the plate on run 1354 to have the serial number 00001. This seamed to fix our problem but I only created the new run with well A01 occupied to ensure the errors we were experiencing went away, which they did. I then went to update run 1358 and this is where I am seeing the updated wells being pooled into the final well on the run.

We will need to identify the conditions causing the above and reproduce the error locally. Following this we should fix the bug. It might be useful to get a dump of prod /training to help reproduce the conditions.
If this is not reproducible after a time boxed period [1-2 days] we can go back to Craig and potentially close this issue.
I have added this to the UI as I suspect the UI is sending or receiving the service data incorrectly however this may not be the case and it may be a service issue.

**To Reproduce**
Steps to reproduce the behaviour:
1. See above

**Expected behaviour**
The wells should stay in their original position and not shift.

**Additional context**
Contacts: Craig C

**Comment by TWJW-SANGER** [comment]
@BenTopping  I'm going to Archive this one. We can bring it back again if needed.

**Comment by BenTopping** [comment]
Yep fine by me, we haven't had any other reports so safe to archive.

**Comment by BenTopping** [comment]
No other incidents reported in over a year. Closing as not planned / can't reproduce.
